dev/faq/data-migration.md

```diff
@@ -22,3 +22,10 @@ Because DM has the feature of concurrently replicating data to the downstream in
 The `driver: bad connection` error indicates that anomalies have occurred in the connection between DM and the upstream TiDB database (such as network failure, TiDB restart and so on) and that the data of the current request has not yet been sent to TiDB at that moment.
 
 When this type of error occurs in the current version, use `stop-task` to stop the task and then use `start-task` to restart the task. The automatic retry mechanism of DM will be improved later.
+
+## What can I do when a replication task is interrupted with failing to get or parse binlog error `get binlog error ERROR 1236 (HY000), binlog checksum mismatch, data may be corrupted` etc. returned?
+
+This error may occur during the incremental replication process of the DM if the binlog file in the upstream exceeds 4 GB, and the DM encounters a replication interruption in the processing of the file (including interruption caused by abnormalities in common pause or stop tasks). It is because that the DM needs to store the replicated binlog position. However, MySQL officially uses unit32 to store it, so the offset value of binlog position of the part exceeding 4 GB overflows and a wrong binlog position is stored. After the task or DM-worker is restarted, this wrong binlog position is used to re-parse the binlog or relay log. In this case, you need to follow these steps to manually recover replication:
+
+- First, determine whether the error occurs in the relay log write or the binlog replication/syncer unit replication(by the component information in the log error message). If the error occurs in the relay log module and the breakpoints saved by the binlog replication/syncer unit are correct, you can first stop the task and the DM-worker, then set the binlog position of the relay meta to `4`, and restart the DM-worker to re-pull the relay log. If the relay log writes, the replication will automatically continue from the breakpoint after restarting the task.
```

Comment:
```suggestion
- First, determine whether the error occurs in the relay log write or the binlog replication/syncer unit replication(by the component information in the log error message). If the error occurs in the relay log module and the checkpoints saved by the binlog replication/syncer unit are correct, you can first stop the task and the DM-worker, then set the binlog position of the relay meta to `4`, and restart the DM-worker to re-pull the relay log. If the relay log writes, the replication will automatically continue from the checkpoint after restarting the task.
```

Reply:
@juliezhang1112 [Ref link from docs-cn](https://pingcap.com/docs-cn/v3.0/reference/tools/tidb-lightning/checkpoints/) like that in TiDB Lightning

Reply:
Thanks for the reference! @lilin90